active/AnimatedIcon/AnimatedIconSpec.md

```diff
@@ -0,0 +1,282 @@
+
+# Animated Icon
+
+## Background
+
+Xaml has an
+[IconElement](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.IconElement)
+base type today with derived types like
+[BitmapIcon](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.BitmapIcon)
+and [SymbolIcon](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.SymbolIcon).
+Since it's an element it can be used in the UI tree, such as in a 
+[StackPanel](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.StackPanel).
+
+All of these icons are static though, and developers do not have a straightforward way to
+add an animated icon in controls where they can use a static icon.
+There is also a desire to have an animated icon respond to user interactivity via state changes to either draw
+attention to an action a user can take or to add user delight.
+
+The [AnimatedVisualPlayer](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer)
+element can be used to create animated (Lottie-based) icons,
+but to have the animation be driven by the states of and interactions with a control,
+developers may need to write a lot of custom code.
+
+Also, AnimatedVisualPlayer does not inherit from IconElement so it cannot be used in places that
+need one, like 
+[NavigationViewItem.Icon](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.NavigationViewItem.Icon)
+or in a button.
+
+This spec introduces an AnimatedIcon IconElement which reduces, or ideally eliminates, the amount of code-behind needed to
+implement most scenarios, and it can be used in WinUI controls that support an IconElement.
+
+# How to design animations for AnimatedIcon
+
+## Creating animated content with Lottie 
```

Comment:
Is this different than how you create Lottie content for AVP? If not, can we just point to what's there?

Reply:
No this is not a different process so I can update this to point to the information for AVP.

Reply:
`will include the markers needed for animated icon to map the correct state transitions to the correct animation segments. It will also contain the themable color properties you can set in code.`

This part is different, although the steps you take as a developer (assuming the json file is formatted with animated icon in mind ie the markers were added) are the same.

Reply:
Right the designer has different steps for AnimatedIcon compared to AVP but the steps the developer needs to do to incorporate the Lottie are the same. That's a good distinction that I will make clearer.

Reply:
updated.